plugins/bazaar/api-report.md

```diff
@@ -62,6 +64,7 @@ export const SortView: (props: SortViewProps) => React_2.JSX.Element;
 export type SortViewProps = {
   fullWidth?: boolean;
   fullHeight?: boolean;
+  codename: string;
```

Comment:
```suggestion
  codename?: string;
```

if you make this optional then you could mark the changeset as `patch`

Reply:
Thanks for the sugestion